backend/src/defence.ts

```diff
@@ -279,6 +282,12 @@ async function detectTriggeredDefences(message: string, defences: Defence[]) {
 	return combineDefenceReports(singleDefenceReports);
 }
 
+// detects triggered defences in bot output and blocks the message if necessary
+function detectTriggeredOutputDefences(message: string, defences: Defence[]) {
+	const singleDefenceReports = [detectFilterBotOutput(message, defences)];
+	return combineDefenceReports(singleDefenceReports);
```

Comment:
Might seem silly to combine just one report, but I like this for two reasons:

1. The return type of `detectFilterBotOutput` is different from that of `combineDefenceReports`.
2. It's consistent with the `detectTriggeredInputDefences` method.